routes/handlers/event.js

```diff
@@ -12,9 +12,9 @@ const OPTS_SCHEMA = Joi.object().keys({
 });
 
 exports.get = function (opts, next) {
-  Joi.validate(opts, OPTS_SCHEMA, function (err, query) {
+  Joi.validate(opts.query, OPTS_SCHEMA, function (err, query) {
     if (err) handleError(next, 'validating', err);
-    const filter = createFilter(query.$filter);
+    const filter = createFilter(query.$filter) || opts.params;
```

Comment:
And this is why we code review. Yep, doesn't need to be there. It was a relic from when I was trying to figure out how it all worked. Removed in the latest commit.